File: order-ms/src/main/java/com/nexeum/orderms/dto/response/ProductItem.java
package com.nexeum.orderms.dto.response;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class ProductItem {
    private Long id;
    private Long productId;
    private String productName;
    private Long productQuantity;
    private BigDecimal totalPrice;
    private BigDecimal unitPrice;
    private String productImageUrl;
}

File: order-ms/dto/response/ProductItem.cs
using System;

namespace OrderMs.Dto.Response
{
    public class ProductItem
    {
        public long Id { get; set; }
        public long ProductId { get; set; }
        public string ProductName { get; set; }
        public long ProductQuantity { get; set; }
        public decimal TotalPrice { get; set; }
        public decimal PricePerUnit { get; set; }
        public string ProductImageUrl { get; set; }
    }
}